### PR TITLE
Added test cases related to Compositor plugin

### PR DESCRIPTION
**Pull request by sandeep-vedam**
Following are the tests that are added in this PR : 
Compositor_Resolution_001.test.js 
Compositor_Resolution_002.test.js 
Compositor_Resolution_003.test.js 
Compositor_Resolution_004.test.js 
Compositor_Resolution_005.test.js 
Compositor_Resolution_006.test.js 
Compositor_Resolution_007.test.js 
Compositor_Resolution_008.test.js 
Compositor_Resolution_009.test.js 
Compositor_Resolution_010.test.js 
Compositor_Resolution_011.test.js 
Compositor_Resolution_012.test.js

Following is the functionality that is not working :
- All Resolutions except 720p60
- Error 22

Note : For the above API's, Response from thunder is not as per the Compositor documentation

### CODE REVIEW

File: src/tests/compositor/Compositor_Resolution_001.test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  description: 'Sets the resolution to 480i and checks whether the same resolution is set or not',
+  context: {
+    resolution: '480i',
+  },
+  steps: [
+    {
+      description: 'Deactivate Netflix Plugin and check deactivated or not',
+      test: pluginDeactivate,
+      params: constants.netFlixPlugin,
+      assert: 'deactivated',
```
Comment (wouterlucas) [inline src/tests/compositor/Compositor_Resolution_001.test.js]: I wouldn't use Netflix for a test here.

File: src/tests/compositor/Compositor_Resolution_001.test.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    },
+    {
+      description: 'Activate Netflix Plugin and check resumed or not',
+      test: pluginActivate,
+      params: constants.netFlixPlugin,
```
Comment (wouterlucas) [inline src/tests/compositor/Compositor_Resolution_001.test.js]: Maybe just use WebKit Browser?

**Review by wouterlucas — CHANGES_REQUESTED** [review]

File: src/tests/compositor/Compositor_Resolution_001.test.js (hunk 1 above)
Comment (sandeep-vedam) [reply src/tests/compositor/Compositor_Resolution_001.test.js]: Updated code as per review comment and used Webkitbrowser

**Review by wouterlucas — APPROVED** [review]

**Comment by wouterlucas** [comment]
approved but has conflicts in commonFunctions

**Pull request merged**